internal/mode/static/nginx/runtime/manager.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	pidFile        = "/etc/nginx/nginx.pid"
+	pidFile        = "/etc/nginx/conf.d/nginx.pid"
```

Comment:
let's move it somewhere else. how about `/var/run/nginx` (a new volume) ? when control plane starts, it will clear `conf.d` folder, removing the pid. If NGINX starts before control plane or control plane container is restarted, we will loose that file.
https://github.com/nginxinc/nginx-kubernetes-gateway/blob/0c3d74af84af77c1b46fb03c4239219871107db2/internal/mode/static/manager.go#L151-L154